meta-balena-common/recipes-core/initrdscripts/files/rootfs

```diff
@@ -76,5 +76,5 @@ rootfs_run() {
     done
 
     # Move the /run mountpoint to the rootfs mountpoint
-    mount -n -o move /run "${ROOTFS_DIR}/run"
```

Comment:
Hi @alexgg remounting `/run` to `${ROOTFS_DIR}/run` seems like the thing you want to do in general. Putting it under `/tmp` is quite unintuitive and I would say will bite us in the future. Is there a way to do this conditionally? Something like
```bash
if corner_case
then
  mount -n -o move /run "${ROOTFS_DIR}/tmp
else
  mount -n -o move /run "${ROOTFS_DIR}/run
fi
```

Reply:
hi @mtoman I did ponder about this myself and my first PR actually took this approach. However, I dislike the corner case approach as it means that specific path is used much less and more likely to have undetected issues.
With this approach both the corner case (first boot after UUID regeneration) and other boots have the same path, and any problems will be apparent and not hidden.
About the "mounting under /tmp" is intrusive. could you please extend? The only runtime difference I see is that there is a new initramfs folder with information passed on from the initramfs.
Having said that, I would also preferred the initial approach of using a distinct /run, but the alternative of adding a HUP hook to pre-populate it when hupping is worse in my opinion.

Reply:
@mtoman as discussed, I have added a comment for clarity, please review